### PR TITLE
Forms Automated Test - LPS-132466 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 1]
```diff
@@ -798,6 +799,14 @@ definition {
 		}
 	}
 
+	macro viewOptionRowName{
+		AssertElementPresent(
+			key_fieldName = "${fieldTitle}",
+			key_rowNumber = "${rowNumber}",
+			key_optionName = "${optionName}",
+			locator1="FormFields#GRID_OPTIONS_OPTION_ROW_NAME");
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: @leilanyaragao can you please sort the parameters alphabetically?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 2]
```diff
@@ -175,14 +175,62 @@ definition {
...
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
 
+		FormFields.viewOptionRowName(
+			fieldTitle = "Grid",
+			rowNumber = "1",
+			optionName = "Option 2");
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase]: @leilanyaragao can you please sort the parameters alphabetically?

**Review by carolmariaabb — COMMENTED** [review]
Great job @leilanyaragao !

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by CalumR23** [comment]
Pushed to BChan here [Pull Request](https://github.com/CalumR23/liferay-portal/pull/3)

**Pull request closed without merging**